frontend/src/GlobalStyles.js

```diff
@@ -4,16 +4,17 @@ export default createGlobalStyle`
 
 * {
     box-sizing: border-box;
-    font-family: "Open Sans", sans-serif;
+    font-family: 'Roboto', sans-serif;
+}
+
+:root {
+    --primary-green: #4C6A28;
```

Comment:
Instead of naming CSS variables `green` or `brown`, try to find a name which tells you where you use it. Imagine that you have two themes. A theme could change the value of `--primary-green` to `red` (example). Now, the name of the variable doesn't make sense anymore.
I would name it `--color-primary` or `--color-text` (or whatever naming convention you prefer).

Reply:
Your explanation sounds reasonable. I will rethink my color variables in general.